### PR TITLE
Update microos script to support .zst file

### PR DESCRIPTION
**Pull request by pyy**
simple way without third part python library.

### CODE REVIEW

File: scripts/microos [hunk 1]
```diff
@@ -54,11 +55,17 @@ def update_group(group):
 
 
 def read_and_parse(url):
-    with urllib.request.urlopen(url) as resp:
-        if url.endswith(".gz"):
-            return ET.parse(gzip.open(resp)).getroot()
-        else:
-            return ET.parse(resp).getroot()
+    if url.endswith(".zst"):
+        os.system(f"wget -O primary.xml.zst {url} && unzstd --rm primary.xml.zst")
```
Comment (aplanas) [inline scripts/microos]: I think that the correct what is not to call `wget` nor `unzstd`. The python code is already downloading the `primary.xml.zst` with `urllib.request.urlopen()`. We can add a new dependency: [zstandard](https://pypi.org/project/zstandard/) and use it to decompress it

**Review by aplanas — CHANGES_REQUESTED** [review]
Thanks a lot for this contribution

**Comment by pyy** [comment]
Hello, thanks for your suggestions. Here is the change.
Now it is using `ET.fromstring()`, is it okay? I don't know how to let zstandard return a file object like `gzip.open()`

Here is another way to consider: https://github.com/openSUSE/openSUSE-release-tools/blob/54075b05b13b59086abf167ddfa7fcca3eba78d7/repo2fileprovides.py#L17

**Review by aplanas — APPROVED** [review]
Both options are OK, so this one LGTM.

Thanks!!

**Pull request merged**